### PR TITLE
Set up resource.limits for thanosQuerier on build01|2

### PR DESCRIPTION
**Pull request by hongkailiu**
Follow up https://coreos.slack.com/archives/C01K2ULGE1H/p1611064054016500

Screen Shot 2021-01-19 at 9 26 00 AM

/cc @openshift/openshift-team-developer-productivity-test-platform @lilic

### CODE REVIEW

File: clusters/build-clusters/01_cluster/openshift-monitoring/cluster-monitoring-config_configmap.yaml [hunk 1]
```diff
@@ -19,3 +19,7 @@ data:
          resources:
            requests:
              storage: 10Gi
+    thanosQuerier:
```
Comment (alvaroaleman) [inline clusters/build-clusters/01_cluster/openshift-monitoring/cluster-monitoring-config_configmap.yaml:22]: Can we add a comment with the corresponding BZ so this doesn't stay here forever?

Comment (hongkailiu) [reply clusters/build-clusters/01_cluster/openshift-monitoring/cluster-monitoring-config_configmap.yaml:22]: Done.

File: clusters/build-clusters/01_cluster/openshift-monitoring/cluster-monitoring-config_configmap.yaml [hunk 2]
```diff
@@ -19,3 +19,8 @@ data:
...
+      # workaround before we have a fix for https://bugzilla.redhat.com/show_bug.cgi?id=1906496
+      resources:
+        limits:
+          memory: 500Mi
```
Comment (s-urbaniak) [inline clusters/build-clusters/01_cluster/openshift-monitoring/cluster-monitoring-config_configmap.yaml:26]: maybe let's set this to 1GB, I think we are indeed idling at ~600MB

Comment (hongkailiu) [reply clusters/build-clusters/01_cluster/openshift-monitoring/cluster-monitoring-config_configmap.yaml]: Done.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**